src/main/java/com/gmail/chernii/oleksii/entity/Skill.java

```diff
@@ -0,0 +1,40 @@
+package com.gmail.chernii.oleksii.entity;
+
+import lombok.Data;
+import lombok.extern.log4j.Log4j;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Created by Space on 16.04.2019.
+ */
+@Data
+@Log4j
+@Entity
+@Table(name = "skills")
+public class Skill {
+    @Id
+    @Column
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column
+    private String level;
```

Comment:
Is better to use enum of even a class for level. 
You could represent junior like j or jun or beginner, or junior engineer or something like this. 
It would be difficult to manage.